librz/core/disasm.c

```diff
@@ -5905,58 +5982,66 @@ RZ_API int rz_core_print_disasm_instructions_with_buf(RzCore *core, ut64 address
 	return len;
 }
 
-static bool handle_backwards_disasm(RzCore *core, int *nb_opcodes, int *nb_bytes) {
-	if (!*nb_opcodes && !*nb_bytes) {
+/**
+ * \brief Handle negative \p pn_opcodes and \p pn_bytes
+ * 	  and seek new offset
+ * 	  and read to core->block
+ *
+ * if nopcodes < 0 {
+ * 	nopcodes = abs(nopcodes),
+ * 	nbytes = nbytes ? min(abs(nbytes), RZ_CORE_MAX_DISASM)
+ * 			: max(core->blocksize, min(abs(nbytes), RZ_CORE_MAX_DISASM))
+ * 	offset = rz_core_prevop_addr_force(...)
+ * } else {
+ * 	offset = core->offset + (nbytes>=0 ? 0 : -nbytes)
+ * 	nopcodes = nopcodes
+ *	nbytes = nbytes ? min(abs(nbytes), RZ_CORE_MAX_DISASM)
+ *			: max(core->blocksize, min(abs(nbytes), RZ_CORE_MAX_DISASM))
+ * }
+ *
```

Comment:
Purpose of documentation is that i want to know how to use an API without reading its implementation, having this code does not help IMHO.